ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/db/ResourceDao.kt

```diff
@@ -32,4 +33,11 @@ interface ResourceDao {
         WHERE a.articleUri = :uri
         """)
     fun getAllForArticle(uri: Uri): List<Resource>
+
+    @Query("""
+        SELECT r.*
+        FROM resources AS r JOIN articleResources AS a ON a.resourceUri = r.uri
+        WHERE a.articleUri = :uri
+        """)
+    fun getAllLiveForArticle(uri: Uri): LiveData<List<Resource>>
```

Comment:
you shouldn't need this LiveData version of the db query

Reply:
Not using the Live Data will make me have to call the data in a background thread.  If you want me to make that change I will.